howto/components-scopes/README.md

```diff
@@ -0,0 +1,66 @@
+# Scope components to be used by one or more applications
+
+There are two things to know about Dapr components in terms of security and access.
+First, Dapr components are namespaced. That means a Dapr runtime instance can only access components that have been deployed to the same namespace.
+
+Although namespace sounds like a Kubernetes term, this is true for Dapr not only on Kubernetes.
+
+## Namespaces
+Namespaces can be used to limit component access to particular Dapr instances.
+
+### Example of component namespacing in Kubernetes
+
+Let's consider the following component in Kubernetes:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore
+  namespace: production
+spec:
+  type: state.redis
+  metadata:
+  - name: redisHost
+    value: redis-master:6379
+```
+
+In this example, the Redis component is only accessible to Dapr instances running inside the `production` namespace.
+
+### Example of component namsespacing in self-hosted mode
+
+In self hosted mode, a developer can specify the namespace to a Dapr instance by setting the `NAMESPACE` environment variable.
+If the `NAMESPACE` environment variable is set, Dapr will not load any component that does not specify the same namespace in its metadata.
```

Comment:
How is this set and is this a general name? Should this not be called "DAPR-NAMESPACE "?

Reply:
It's a general name, Kubernetes and a few other projects use it without prefixes. In the future if people have different wants, we can look into changing that.

Reply:
The namespace env var btw is not something new, we had it since 0.1.0